static/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -657,7 +657,7 @@ class GroupDetails extends Component<Props, State> {
   render() {
     const {project, group} = this.state;
     const {organization} = this.props;
-    const isSampleError = group?.tags.some(tag => tag.key === 'sample_event');
+    const isSampleError = group?.tags?.some(tag => tag.key === 'sample_event');
```

Comment:
Is there a type we can check

Reply:
It was the right type. I think this is a null checks issue